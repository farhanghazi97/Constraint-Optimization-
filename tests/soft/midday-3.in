# Testing midday soft constraint
meeting, m1
domain, m1, midday, soft
domain, m1, 4pm, hard
domain, m1, thu, hard
