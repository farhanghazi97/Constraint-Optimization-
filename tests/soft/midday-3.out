m1:thu 4pm
cost:4
